Tests/SonarScanner.MSBuild.PreProcessor.Test/CacheProcessorTests.cs

```diff
@@ -49,7 +49,7 @@ public class CacheProcessorTests
         [TestMethod]
         public void Constructor_NullArguments_Throws()
         {
-            var server = Mock.Of<ISonarQubeServer>();
+            var server = Mock.Of<ISonarWebService>();
```

Comment:
`webService`

Reply:
As discussed offline, this would need to be changed all over the production code too. And `server` is not that bad